man/en/sos-report.1

```diff
@@ -180,6 +182,17 @@ compression-type file extension for example ".zip". ".1", ".gz" etc.).
 This also affects \--all-logs. The date string will be padded with zeros
 if HHMMSS is not specified.
 .TP
+.B \--skip-commands COMMANDS
+A comma delimited list of commands to skip execution of, but still allowing the
+rest of the plugin that calls the command to run. This will generally need to
+be a regular expression, and unix-style shell regex syntax is supported. For
+example, to skip all 'foobar' commands, use '--skip-commands foobar*'
+.TP
+.B \--skip-files FILES
+A comma delimited list of files or filepath regexes to skip collection of, but
+still allowing the rest of the plugin that tries to collect the files to run.
+These values may be explicit filepaths, or unix-style regexes.
+.TP
```

Comment:
Looking at the decision code later in the patch we're using `fnmatch.fnmatch` for the pattern matching - this does UNIX style file name pattern matching using glob syntax so I think it's best to avoid calling these regular expressions at all. The doc text for the module says this in the introduction: "This module provides support for Unix shell-style wildcards, which are not the same as regular expressions (which are documented in the re module)."

Reply:
That's a very fair point.

I am not 100% sold on using `fnmatch` here; however it makes perfect sense for `--skip-files` and we don't want to introduce 2 different styles for very similar options (i.e. using `re` for `--skip-commands`). But, if we do go down that route, it should definitely be as clear as possible in the docs.

Reply:
Yeah agree - I think the way you've done it is likely to be the most intuitive for admins; I think it would be confusing to have e.g. `--skip-files /etc/systemd*` but then insist on `--skip-commands system.*` - just a case of having the docs clear. I'm not a huge fan of the term "glob" in docs (even though it's part of POSIX) because it's a bit obscure - I think "shell style pattern matching" is one of the phrases I've used in the past?

Reply:
Updated the phrasing to "UNIX shell-style wildcards" which I think is sufficient.